news/_posts/core-weekly/2018-10-31-coreweekly-week-43-2018.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 43 of 2018"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2018-10-31 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 22nd to Sunday 28th of October 2018.
+
+![Core Weekly banner](/assets/images/2017/04/core_weekly_banner.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Last week, part of the PrestaShop team's developers attended the PHP Forum in Paris. It is a good event to discover what's new with the technology and learn new skills. However, it has an impact on the productivity: less pull requests have been merged than usual.
```

Comment:
However, it has had an impact on productivity: fewer pull requests have been merged than usual.